blockchain/go.mod

```diff
@@ -12,7 +12,10 @@ require (
 	github.com/decred/dcrd/dcrec/secp256k1 v1.0.2
 	github.com/decred/dcrd/dcrutil/v2 v2.0.0
 	github.com/decred/dcrd/gcs v1.1.0
```

Comment:
Can't you remove gcs@v1.1.0 from here as well?

Reply:
Yes. I should've run tidy on it.  Good catch!

Reply:
Done.